docs/actors/memory_and_cpu.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Memory and CPU
+description: Learn about your actors' memory and processing power requirements, their relationship with Docker resources, and minimum requirements for different use cases.
+menuWeight: 7.5
+paths:
+# NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
+    - actors/memory-and-cpu
+---
+
+# Memory and CPU
+
+[Actors]({{@link actors.md}}) run in
+[Docker containers](https://www.docker.com/resources/what-container),
+which have a [limited amount of resources](https://phoenixnap.com/kb/docker-memory-and-cpu-limit). When starting, the actor needs to be allocated a certain share of those resources, such as how much CPU "horsepower" it will be able to use.
+
+![Setting an actor's memory]({{@asset actors/images/memory-settings.png}})
+
+By assigning an actor's **memory** capacity, you also assign it a proportional amount of **CPU power**. Every CPU core corresponds to 4 GB of memory. So, 512 MB = 1/8th of a core, 1 GB = 1/4 of core, 8 GB gives you 2 cores, and so on.
+
+In addition to CPU power, the memory allocation also influences [disk size and Input/Output (I/O) limits](https://docs.docker.com/config/containers/runmetrics/#metrics-from-cgroups-memory-cpu-block-io).
```

Comment:
We are limiting disk size, but I/O throttling is still WIP. ATM we are not enforcing it. So I would leave that part out for now. We can update the docs when we implement it.

Also I would loose the link. You do not have it in CPU or Memory section either. Besides it points to runtime metrics not the I/O constraints.

Reply:
Please keep disk size here (it's 2x memory right @mtrunkat ?) But loose I/O limits.

Reply:
🤦  thanks!